### PR TITLE
add python script compilation with pyinstaller in install step

### PR DESCRIPTION
**Pull request by jmkerloch**
- add `pyinstaller `search in `CMakeList.txt`
- if `pyinstaller `found, compile python scripts, otherwise copy pythons scripts in installer
- update GitHub action CI to install `pyinstaller`
- add `requirements.txt` for python scripts dependency (for now only `pyinstaller`)

### CODE REVIEW

File: .github/workflows/dockerHub.yml [hunk 1]
```diff
@@ -22,6 +22,15 @@ jobs:
...
     - uses: actions/checkout@v2
     
+    - name: Set up Python
+      run: |
+          apt-get install python3-pip -y
+                    
+    - name: Install dependencies
+      run: |
+          python3 -m pip install --upgrade pip
+          pip install -r src/src_python/requirements.txt
```
Comment (flomnes) [inline .github/workflows/dockerHub.yml:32]: pip3

File: .github/workflows/releases.yml [hunk 2]
```diff
@@ -24,6 +24,16 @@ jobs:
     steps:
     - uses: actions/checkout@v2 
     
+    - name: Set up Python
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.6
```
Comment (flomnes) [inline .github/workflows/releases.yml:30]: python 3.6 no longer receives support. Use 3.8 or 3.9 ?

[table of versions](https://en.wikipedia.org/wiki/History_of_Python#Table_of_versions)

**Review by flomnes — CHANGES_REQUESTED** [review]

File: .github/workflows/dockerHub.yml (hunk 1 above)
Comment (jmkerloch) [reply .github/workflows/dockerHub.yml]: ok changed

File: .github/workflows/releases.yml (hunk 2 above)
Comment (jmkerloch) [reply .github/workflows/releases.yml]: use 3.8

**Review by sgatto — APPROVED** [review]
OK pour moi

File: .github/workflows/releases.yml (hunk 2 above)
Comment (sgatto) [reply .github/workflows/releases.yml]: ok

**Review by flomnes — APPROVED** [review]

**Pull request merged**